File: Baekjoon/2단계 조건문/오븐 시계.py
hour, minute = map(int, input().split())
timer = int(input())

hour += timer // 60
minute += timer % 60

if minute >= 60:
    hour += minute // 60
    minute %= 60
if hour > 23:
    hour = hour % 24

print(f'{hour} {minute}')

File: Baekjoon/2단계 조건문/js/오븐 시계.js
const fs = require('fs');

const [input1, input2] = fs.readFileSync('/dev/stdin').toString().split('\n');

let [hour, minute] = input1.split(' ').map(Number);
const totalCookTime = Number(input2);

const cookHour = parseInt(totalCookTime / 60, 10) % 24;
const cookMinute = totalCookTime % 60;

hour += cookHour;
minute += cookMinute;

if (minute > 59) {
  minute -= 60;
  hour += 1;
}
if (hour >= 24) hour -= 24;

console.log(`${hour} ${minute}`);
